fastcrypto-vdf/benches/class_group.rs

```diff
@@ -43,18 +41,15 @@ fn class_group_ops(c: &mut Criterion) {
 }
 
 fn qf_from_seed_single<M: Measurement>(discriminant_string: &str, group: &mut BenchmarkGroup<M>) {
```

Comment:
why do we call this "from_seed" while the seed is thread locally randomly generated? (and probably not even with a secure rng?)